assets/xml/objects/object_am.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<Root>
     <File Name="object_am" Segment="6">
-        <Collision Name="object_am_Colheader_000118" Offset="0x118" />
+        <Collision Name="object_am_Colheader_000118" Offset="0x118" /> <!-- Original name is "am_bgdatainfo" -->
```

Comment:
This is "am_" but the collider is "armos_" ? Guess that supports the theory armos was thought of second

Reply:
Yeah, it's really weird. I had to double-check this one to make absolutely sure I was labeling the two of them correctly.